src/App.jsx

```diff
@@ -2,11 +2,57 @@ import './App.css'
 
 function App() {
 
-  return (
-    <>
-        <p>Start</p>
-    </>
-  )
+    const goods = [
+        {
+            id: 0,
+            name: "Laptop",
+            brand: "Dell",
+            price: 799,
+            features: ["16GB RAM", "512GB SSD", "Intel i7"],
+            inStock: true
+        },
+        {
+            id: 1,
+            name: "Smartphone",
+            brand: "Apple",
+            price: 999,
+            features: ["128GB Storage", "A14 Chip", "5G Enabled"],
+            inStock: false
+        },
+        {
+            id: 2,
+            name: "Headphones",
+            brand: "Sony",
+            price: 199,
+            features: ["Noise Cancelling", "Wireless", "20-hour battery life"],
+            inStock: true
+        },
+    ];
+
+
+    return (
+        <div className="item-list">
+            <h1>Items List</h1>
+            <ul>
+                {goods.map((item, index) => (
+                    <li key={index} className="item">
```

Comment:
It would be better to use id as a key instead of index

Reply:
Yes, totally agree!